src/panel/poi/blocks/Block.jsx

```diff
@@ -22,7 +22,11 @@ const Block = ({
       title={tooltip}
       {...rest}
     >
-      <i className={`block-icon icon-${icon}`} />
+      {typeof icon === 'string' ? (
+        <i className={`block-icon icon-${icon}`} />
+      ) : (
+        <div className="u-mr-s">{icon}</div>
```

Comment:
Needed change because this new icon wasn't in the icon font and we prefer using SVG icons now.